h2o-docs/src/product/data-science/algo-params/monotone_constraints.rst

```diff
@@ -9,7 +9,7 @@ Description
 
 A mapping that represents monotonic constraints. Use +1 to enforce an increasing constraint and -1 to specify a decreasing constraint. Note that constraints can only be defined for numerical columns. 
 
-**Note**: In GBM and XGBoost, this option can only be used when the distribution is ``gaussian``, ``bernoulli``, or ``tweedie``.
+**Note**: In GBM and XGBoost, this option can only be used when the distribution is ``gaussian``, ``bernoulli``, ``tweedie`` or ``quantile``.
```

Comment:
you are changing the docs for xgboost as well, is that intentional?

Reply:
Good point, for xgboost, quantile is not supported! Will fix.

Reply:
Fixed. It is now better?